.github/CODEOWNERS

```diff
@@ -1,7 +1,7 @@
 # CODEOWNERS: https://help.github.com/articles/about-codeowners/
 
 # Primary repo maintainers
-*       @rigelrozanski @alexanderbez @jackzampolin @alessio @fedekunze
+*       @alessio @clevinson @ethanfrey
```

Comment:
I don't believe this does what you would like. You should set these codeowners to the 0.39 line. so you three would be code owners for the backport branches only. 

 This won't change anything as the codeowners need to be on the default branch...

Reply:
I originally intended to use the `CODEOWNERS`hip thingy as a way to automatically notify the people listed there on PRs that was meant to be merged against launchpad-specific branches (cause when you are in CODEOWNERS you get the notification automatically every time a PR is opened). But I think you're right, maybe this is not the best way of doing it. Can you help me please?

Reply:
(meanwhile, I've reverted the respective commit)